crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -767,37 +755,18 @@ async fn update_single_agent_info(
     put_local_agent_info(evt_sender.clone(), agent_info_signed.clone()).await?;
 
     // Push to the network as well
-    match network_type {
-        NetworkType::QuicMdns => {
-            // Broadcast only valid AgentInfo
-            if !urls.is_empty() {
-                // Kill previous broadcast for this space + agent
-                let key = [space.get_bytes(), agent.get_bytes()].concat();
-                if let Some(current_handle) = mdns_handles.get(&key) {
-                    mdns_kill_thread(current_handle.to_owned());
-                }
-                // Broadcast by using Space as service type and Agent as service name
-                let space_b64 = base64::prelude::BASE64_URL_SAFE_NO_PAD.encode(&space[..]);
-                let agent_b64 = base64::prelude::BASE64_URL_SAFE_NO_PAD.encode(&agent[..]);
-                //println!("(MDNS) - Broadcasting of Agent {:?} ({}) in space {:?} ({} ; {})",
-                // agent, agent.get_bytes().len(), space, space.get_bytes().len(), space_b64.len());
-                // Broadcast rmp encoded agent_info_signed
-                let mut buffer = Vec::new();
-                rmp_encode(&mut buffer, &agent_info_signed)?;
-                tracing::trace!(?space_b64, ?agent_b64);
-                let handle = mdns_create_broadcast_thread(space_b64, agent_b64, &buffer);
-                // store handle in self
-                mdns_handles.insert(key, handle);
-            }
-        }
-        NetworkType::QuicBootstrap => {
-            kitsune_p2p_bootstrap_client::put(
-                bootstrap_service.clone(),
-                agent_info_signed.clone(),
-                bootstrap_net,
-            )
-            .await?;
-        }
+    {
+        // TODO: "mdns" stuff
+    }
```

Comment:
Putting this as a new comment since the previous seems to have been lost: https://github.com/holochain/holochain/pull/3225/files#r1578112811

We still need to delete or do something with the `// TODO: "mdns" stuff` comment above.

Reply:
This is not where it'd go. Local discovery is going to be handled inside tx5.